### PR TITLE
Minor edits to Password Inline Hook content

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:
- **What's changed?** Minor edits.
- **Is this PR related to a Monolith release?** No.

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -356,10 +356,11 @@ curl -v -X POST \
...
+Creates a user with a [Password Hook](#password-hook-object) object specifying that a Password Inline Hook should be used to handle password verification.
 
-The new user is able to login immediately after activation with the valid password.
-This flow supports migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
+The Password Inline Hook is triggered to handle verification of the end user's password the first time the user tries to log in, with Okta calling the Password Inline Hook to check that the password the user supplied is valid. If the password is valid, Okta stores the hash of the password that was provided and can authenticate the user independently from then on. See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+
+The new user is able to log in immediately after activation with the valid password. This flow supports migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md]: ... in **use** cases where we **want** to allow ....

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Okta defines several different types of Inline Hooks. Each type of Inline Hook m
 | [User Import Inline Hook](/docs/reference/import-hook/)        | Adds custom logic to the user import process.                                   |
 | [SAML Assertion Inline Hook](/docs/reference/saml-hook/)       | Customizes SAML assertions returned by Okta.                                    |
 | [Registration Inline Hook](/docs/reference/registration-hook/) | Customizes handling of user registration requests in Self-Service Registration. |
-| [Password Import Inline Hook](/docs/reference/password-hook/)   | Verifies a user-supplied password during migration of the user to Okta          |
+| [Password Import Inline Hook](/docs/reference/password-hook/)  | Verifies a user-supplied password to support migration of users to Okta.        |
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md]: These are fragments, so you need to remove the periods at the end.

File: packages/@okta/vuepress-site/docs/reference/password-hook/index.md [hunk 3]
```diff
@@ -145,9 +145,9 @@ If the default action sent by Okta in the `action.credential` property of the re
 
 ## Enabling a Password Import Inline Hook
 
-To enable a Password Import Inline Hook, you first need to register your external service endpoint with Okta and configure it as an Inline Hook of type `com.okta.user.credential.password.import`; see [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hook-setup). You can also use the Admin Console to register your external service endpoint and configure the hook by going to **Workflow > Inline Hooks** and clicking `Add Inline Hook`.
+To enable a Password Import Inline Hook, you first need to register your external service endpoint with Okta and configure it as an Inline Hook of type `com.okta.user.credential.password.import`; see [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hook-setup). You can also use the Admin Console to register your external service endpoint and configure the hook by going to **Workflow > Inline Hooks** and clicking **Add Inline Hook**.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/password-hook/index.md]: To enable a Password Import Inline Hook, you first need to register your external service endpoint with Okta and configure it as an Inline Hook of type `com.okta.user.credential.password.import`**. S**ee [Inline Hook Setup].

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 4]
```diff
@@ -356,10 +356,11 @@ curl -v -X POST \
...
+Creates a user with a [Password Hook](#password-hook-object) object specifying that a Password Inline Hook should be used to handle password verification.
 
-The new user is able to login immediately after activation with the valid password.
-This flow supports migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
+The Password Inline Hook is triggered to handle verification of the end user's password the first time the user tries to log in, with Okta calling the Password Inline Hook to check that the password the user supplied is valid. If the password is valid, Okta stores the hash of the password that was provided and can authenticate the user independently from then on. See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md]: ....the user tries to sign in, with Okta calling.....

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: The new user is able to sign in immediately.....

**Pull request merged**